.gitmodules

```diff
@@ -25,3 +21,6 @@
 [submodule "models/CovidMetrics-epiBATS"]
 	path = models/CovidMetrics-epiBATS
 	url = git@github.com:arodloff/autosubmission-CovidMetrics-epiBATS.git
+[submodule "models/ecdc-covid19-admissions"]
```

Comment:
maybe better to call `epiforecasts-admissions` so as not to suggest to anyone this is run by ECDC?